integration/Jenkinsfile

```diff
@@ -35,9 +35,12 @@ pipeline {
                         stage('Cleanup Workspace') {
                             steps {
                                 cleanWs()
-                                sh """
+                                sh '''
                                 echo "Cleaned Up Workspace For Project"
-                                """
+                                docker stop $(docker ps -a -q)
+                                docker rm $(docker ps -a -q)
+                                docker rmi $(docker images -q)
+                                '''
```

Comment:
may be define as a groovy var and then u can reuse it later.

Reply:
fixed